addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -104,7 +104,7 @@ FormRenderer.include({
      * @returns {jQuery.Element}
      */
     _makeChatterContainerTarget() {
-        const $el = $('<div class="o_FormRenderer_chatterContainer"/>');
+        const $el = $('<div class="o_FormRenderer_chatterContainer" data-command-category="mail"/>');
```

Comment:
This is a compatibility layer, prefer moving this into `ChatterContainer` itself.